app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,15 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  height: 70px;
```

Comment:
Why add the height? This restricts content that may be long. Seems unnecessary.

<img width="360" alt="Screenshot 2020-02-19 at 08 21 03" src="https://user-images.githubusercontent.com/861310/74815248-cccf7c80-52f0-11ea-9eab-e61ec71051f7.png">

Reply:
have checked differing lengths of content and indeed the height is not necessary so have removed